config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/loader/ServiceAndPolicyLoaderUtil.java

```diff
@@ -13,6 +14,17 @@
 public class ServiceAndPolicyLoaderUtil {
 
     public static final String HANDLE_DUPLICATE_NAMES = "com.ca.apim.export.handleDuplicateNames";
+    public static final String ANNOTATE_PORTAL_INTEGRATION_ASSERTIONS_PROP = "com.ca.apim.export.migratePortalIntegrationAssertions";
+    private static String ANNOTATE_PORTAL_INTEGRATION_ASSERTIONS = System.getProperty(ANNOTATE_PORTAL_INTEGRATION_ASSERTIONS_PROP);
+
+    public static boolean isAnnotatePortalApisSet() {
+        return ANNOTATE_PORTAL_INTEGRATION_ASSERTIONS != null && "true".equals(ANNOTATE_PORTAL_INTEGRATION_ASSERTIONS.trim());
+    }
+
+    @VisibleForTesting
```

Comment:
Is this required? You can always set the system property prior to running the test.